src/en/installconfig-proxy.md

```diff
@@ -0,0 +1,91 @@
+Title: Proxy Server
+TODO:  Needs to be rewritten for clarity
+
+# Proxy Server
+
+In most cases, the machines deployed in a MAAS installation will use an
+internal proxy server to access the archive. Alternatively, MAAS can be told to
+use an externally configured proxy, or none at all.
+
+As of MAAS 2.0, the default MAAS region controller configuration will create a
+maas-proxy configuration that explicitly allows all of the subnets in the
+configuration to use it for a proxy.
+
+## Assumptions
+
+It is generally assumed that the MAAS region controller is not exposed to
+untrusted networks (there's a firewall between the MAAS region
+controller and any such networks). Prior to MAAS 2.0, the only option if this
+was not the case was to manually edit `/etc/maas/maas-proxy.conf`, or use
+iptables to create a firewall on the host.
+
+As of MAAS 2.0, permission to use the proxy can be managed on a per-subnet
+basis, with the default to allow proxying.
+
+!!! Warning: If your MAAS Region is connected to an untrusted network, you
+should disable that subnet in the proxy, as shown below.
+
+## MAAS Proxy Installation
+
+When you install a Region Controller, maas-proxy will be installed. The
+configuration of the proxy relies on maas-proxy being on the same machine as
+the Region Controller.
+
+### Using an external proxy
+
+If you want to use an external proxy, you can define its URL using the web
+interface by selecting Settings>General and scrolling down to the 'Proxy for
+APT and HTTP/HTTPS' section:
```

Comment:
The proxy is only for APT and not also for HTTP/HTTPS. I've been meaning to open a bug on this for a while. Do we   want to clarify this here?